src/PresentationalComponents/RulesTable/RulesTable.spec.ct.js

```diff
@@ -0,0 +1,642 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from '@cypress/react';
+import RulesTable from './RulesTable';
+import { Provider } from 'react-redux';
+import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations/';
```

Comment:
```suggestion
import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations';
```
because the trailing slash can sometimes lead to problems with package discovery

Reply:
You are correct. removed it